piano.yml

```diff
@@ -282,7 +282,7 @@ Layer:
     <<: *db
     table: |-
         ( SELECT
-            geometry, COALESCE("name:{lang}", name) AS name, '{lang}' AS lang, prio::int, NULL as ldir
+            geometry, COALESCE("name:{lang}", name) AS name, '{lang}' AS lang,iso, prio::int, NULL as ldir, NULLIF(sov, '') as sov
```

Comment:
nit: missing a space before the comma ;)

Reply:
should be corrected by #bf2d01d